Homework.RehabE.

```diff
@@ -0,0 +1,31 @@
+
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package javaapplication1;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author rehabahmed
+ */
+public class JavaApplication1 {
```

Comment:
The class name must match the file name. For example, this file should be named `JavaApplication1.java` in a folder named `javaapplication1` (which matches the `package` name). Also `JavaApplication1` probably isn't a great class name for this class. You may want to use something like `CityNameAnalyzer` or something of that nature.